luigi/batch_notifier.py

```diff
@@ -76,18 +76,19 @@ def _key(self, task_name, family, unbatched_args):
         elif self._config.batch_mode == 'family':
             return family
         elif self._config.batch_mode == 'unbatched_params':
-            param_str = six.u(', ').join(six.u('{}={}').format(*kv) for kv in six.iteritems(unbatched_args))
-            return six.u('{}({})').format(family, param_str)
+            param_str = ', '.join(f'{k}={v}' for k, v in six.iteritems(unbatched_args))
+            return f'{family}({param_str})'
```

Comment:
weren't f-strings added in Python 3.6? If so, are we also removing support for 3.4 and 3.5?

Reply:
Right. Good catch! We shouldn't remove that I think.

Reply:
@dlstadther @honnix 
fixed